docker/config_template.yaml

```diff
@@ -129,7 +129,7 @@ services:
         metrics:
             prometheus:
                 timerType: {{ default .Env.PROMETHEUS_TIMER_TYPE "histogram" }}
-                listenAddress: {{ .Env.PROMETHEUS_ENDPOINT }}
+                listenAddress: {{ .Env.PROMETHEUS_FRONTEND_ENDPOINT }}
         {{- end }}
```

Comment:
This if is part of picking statsd vs prom using env vars: `{{- else if .Env.PROMETHEUS_ENDPOINT }}` - this would mean still setting `PROMETHEUS_ENDPOINT` in addition to the per-service env vars.